t/node/chash-balance.t

```diff
@@ -171,3 +171,81 @@ GET /t
 [{"count":12,"port":"1982"}]
 --- no_error_log
 [error]
+
+
+
+=== TEST 5: set route(three upstream node)
+--- config
+    location /t {
+        content_by_lua_block {
+            local t = require("lib.test_admin").test
+            local code, body = t('/apisix/admin/routes/1',
+                 ngx.HTTP_PUT,
+                 [[{
+                    "uri": "/server_port",
+                    "upstream": {
+                        "key": "remote_addr",
+                        "type": "chash",
+                        "nodes": {
+                            "127.0.0.1:1980": 1,
+                            "127.0.0.1:1981": 0,
+                            "127.0.0.1:1982": 0
+                        }
```

Comment:
need to check the `nodes` wrote into etcd.

Reply:
https://github.com/iresty/apisix/pull/536/files#diff-3d55f969c5c7e4abd188c26251844464R750

fixed